securedrop/journalist_templates/js-strings.html

```diff
@@ -6,4 +6,5 @@
   <div id="select-none-string" hidden>{{ gettext('Select None') }}</div>
   <div id="delete-user-confirm-string" hidden>{{ gettext('Are you sure you want to delete the user {username}?') }}</div>
   <div id="reset-user-mfa-confirm-string" hidden>{{ gettext('Are you sure you want to reset two-factor authentication for {username}?') }}</div>
+  <div id="sources-selected" hidden>{{ gettext('Sources selected: ') }}</div>
```

Comment:
If I'm not mistaken, this is a compromise to accommodate lack of pluralization support in our current solution for managing client-side strings. It may be worth at least minimally augmenting the `get_string` method to support fetching basic singular/plural variants so we can use more human-friendly strings. For any changes to the JI, including admin features, this problem is likely to arise again in future. I understand this may be pushing it for 1.8.0.

Reply:
Yup @rmol and I briefly discussed this. Having some way to properly handle translateable JS strings would be useful, but it's not going to happen for this release.

Reply:
The consensus among Arabic translators was that this usage ("Things: x")  is not optimal, but people would understand.